src/test/java/conformance/NullnessJSpecifyConformanceTest.java

```diff
@@ -25,35 +25,46 @@
 /** A class to run the conformance tests against the EISOP Checker Framework. */
 public final class NullnessJSpecifyConformanceTest {
 
-  /** Directory of the JSpecify Conformance Tests. */
-  private final Path testDir;
-
-  /** Location of the report. */
-  private final Path reportPath;
-
-  /** Directory of the JSpecify Conformance Tests for samples. */
-  private final Path testDirSamples;
-
-  /** Location of the report for samples. */
-  private final Path reportPathSamples;
-
-  /** JSpecify conformance test dependencies. */
-  private final ImmutableList<Path> deps;
-
   /** Options to pass to the checker. */
   private static final ImmutableList<String> TEST_OPTIONS =
       ImmutableList.of("-AassumePure", "-Adetailedmsgtext");
 
-  /** Create a NullnessJSpecifyConformanceTest. */
-  public NullnessJSpecifyConformanceTest() {
-    this.testDir = getSystemPropertyPath("ConformanceTest.inputs");
-    this.reportPath = getSystemPropertyPath("ConformanceTest.report");
-    this.testDirSamples = getSystemPropertyPath("ConformanceTest.samples.inputs");
-    this.reportPathSamples = getSystemPropertyPath("ConformanceTest.samples.report");
-    this.deps =
-        Splitter.on(":").splitToList(System.getProperty("ConformanceTest.deps")).stream()
-            .map(dep -> Paths.get(dep))
-            .collect(toImmutableList());
+  /** Compiler Provided Default Ctor is fine */
```

Comment:
```suggestion
```

Reply:
This comment is for the default constructor... the javadoc will either get lost or merged into the next method, neither of which is what is desired.